src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -188,6 +188,25 @@ public void ContentType_SetThenGetEmptyValue_ExpectNull(Uri remoteServer)
             });
         }
 
+        [Fact]
+        public async Task HttpWebRequest_SetHostHeader_ContainsPortNumber()
+        {
+            await LoopbackServer.CreateServerAsync(async (server, uri) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(uri);
+                string host = uri.Host + ":" + uri.Port;
+                request.Host = host;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+
+                await server.AcceptConnectionAsync(async connection =>
+                {
+                    // Skip status line.
+                    await connection.Reader.ReadLineAsync();
+                    Assert.Contains(host, await connection.Reader.ReadLineAsync());
```

Comment:
nit: For better readability, add blank line above this to separate out the "// Skip status line" comment and code from this line of code. Or move the comment to end of ReadLineAsync().

Reply:
Will fix.